src/components/modals/tx/txConfirmedFlashbotsModal.tsx

```diff
@@ -0,0 +1,44 @@
+import { FC } from 'react';
+import { Button, Flex, Text } from '@chakra-ui/react';
+import { IoArrowUpCircleOutline } from 'react-icons/io5';
+import {
+  Modal,
+  ModalBody,
+  ModalCloseButton,
+  ModalContent,
+  ModalFooter,
+  ModalHeader,
+  ModalOverlay,
+} from '@chakra-ui/modal';
+
+type Props = {
+  onClose: () => void;
+  message?: string;
+};
+
+const TxConfirmedFlashbotsModal: FC<Props> = ({ onClose }) => {
+  return (
+    <Modal isOpen={true} onClose={onClose}>
+      <ModalOverlay />
+      <ModalContent borderRadius='xl'>
+        <ModalHeader textAlign='center'>Transaction successful!</ModalHeader>
+        <ModalCloseButton />
+        <ModalBody textAlign='center' py={8}>
+          <Flex justifyContent='center' color='white' pb={2}>
+            <IoArrowUpCircleOutline fontSize='80px' />
+          </Flex>
+          <Text color='gray.200'>
+            Your transaction was successfully completed via Flashbots!
```

Comment:
Minor but in unstakeAndClaim you are passing a message of 'success', you could have passed 'Your transaction was succesfully completed via Flashbots!' and used the message prop 

```suggestion
            {message}
```

Reply:
Another great suggestion! Will be in next commit